react/components/QuantityStepper.tsx

```diff
@@ -139,12 +139,14 @@ const QuantityStepper: VFC<Props> = ({
     <div className="flex">
       <button
         className={classnames(
+          styles.quantityButton,
           'pa4 ba br2 br--left flex items-center justify-center',
           {
             'c-muted-1 b--muted-4 hover-b--muted-3 bg-muted-5 hover-bg-muted-4 pointer': !disabled,
             'bg-muted-5 c-muted-3 b--muted-4': disabled,
           }
         )}
+        data-quantity-action="decrease"
```

Comment:
why not create another css handle for this?

Reply:
@lucasecdb  I don´t create because we can use the selector [data-*}. But, I can add with modifiers